src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/custom.py

```diff
@@ -1077,8 +1077,119 @@ def update_policy_setdefinition(cmd, policy_set_definition_name, definitions=Non
     return policy_client.policy_set_definitions.create_or_update(policy_set_definition_name, parameters)
 
 
+def _register_rp(cli_ctx, subscription_id=None):
```

Comment:
Why is this in here?  @yugangw-msft @lmazuel correct me if I'm wrong but msrest will automatically handle this, correct?

Reply:
You can look at the discussion here - https://github.com/Azure/azure-cli-extensions/pull/62